Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ghcr.io/bento-platform/bento_base_image:python-debian-2022.10.11
+
+# TODO: change USER
+USER root
+
+RUN apt install gcc libffi-dev -y
+
+RUN mkdir -p /drs/bento_drs && \
+    mkdir /wes;
+
+RUN echo "Building DRS in Production Mode";
+
+WORKDIR /drs/bento_drs
+COPY . .
+
+WORKDIR /drs/bento_drs
+RUN mkdir -p /drs/bento_drs/data/obj && \
+    mkdir -p /drs/bento_drs/data/db;
+RUN ["pip", "install", "-r", "requirements.txt"]
```

Comment:
it's better if this comes before `COPY . .` to preserve cache

so a general flow might look like

```dockerfile
WORKDIR /drs/bento_drs
 
# Install dependencies
COPY requirements.txt requirements.txt
RUN pip install -r requirements.txt

# Copy app code
COPY . .
```